dev/com.ibm.ws.jca_fat_derbyra/publish/servers/com.ibm.ws.jca.fat.derbyra/bootstrap.properties

```diff
@@ -1,4 +1,4 @@
-com.ibm.ws.logging.trace.specification=*=info:logservice=all:rarInstall=all:WAS.j2c=all:test=all:web.*=all:com.ibm.ws.config.xml.internal.MetaTypeRegistry=all:com.ibm.ws.connectionpool.monitor.*=all
+com.ibm.ws.logging.trace.specification=*=info:app.manager=all=enabled:logservice=all:rarInstall=all:WAS.j2c=all:test=all:web.*=all:com.ibm.ws.config.xml.internal.MetaTypeRegistry=all:com.ibm.ws.connectionpool.monitor.*=all
```

Comment:
Did you mean to check in app manager trace enablement?  There was recently an effort to prune the amount of trace recorded by tests, and app management was one of the main offenders, so we should be cautious about adding it